practice/react-practice-one/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,46 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import components */
+import Button from '../Buttons/Button';
+
+/* Import CSS */
+import './pagination.css';
+
+const Pagination = ({ currentPage, totalRecords, pageLimit, onPageChange }) => {
+  const totalPages = Math.ceil(totalRecords / pageLimit);
+
+  const handleClick = (page) => {
+    if (page !== currentPage) {
+      onPageChange(page);
+    }
+  };
+
+  const renderButtons = () => {
+    const buttons = [];
+
+    for (let i = 1; i <= totalPages; i++) {
```

Comment:
@JennyDoJD how about if we have 100 pages?

Reply:
@JennyDoJD did you check my comment?

Reply:
I got it. I'm fixing your comment.

Reply:
![image](https://github.com/user-attachments/assets/546a867d-e25f-40ad-8a28-6417435ff9b3)